tests/next_tests/unit_tests/program_processor_tests/runners_tests/transformation_tests/__init__.py

```diff
@@ -0,0 +1,14 @@
+# GT4Py - GridTools Framework
```

Comment:
The module name should contain "dace". We could create a subpackage for it and move my existing tests there too:
```
tests/next_tests/unit_tests/program_processor_tests/runners_tests/dace/transformation_tests/__init__.py
tests/next_tests/unit_tests/program_processor_tests/runners_tests/dace/test_gtir_to_sdfg.py (was test_dace_fieldview.py)
```